test/org/sagebionetworks/bridge/stormpath/StormpathAccountDaoMockTest.java

```diff
@@ -263,6 +273,19 @@ public void authenticate() {
     }
 
     @Test
+    public void getAccountWithEmail() {
+        mockAccountWithoutHealthCode();
+        doReturn(accountList).when(directory).getAccounts(any(AccountCriteria.class));
+        
+        dao.getAccountWithEmail(study, "email@email.com");
```

Comment:
Is it worth putting data in accountList and verify that it's correctly passed back? Also, do we need to verify that we're passing in study correctly and getting the right directory?